integration/test/README.md

```diff
@@ -44,19 +26,37 @@ application to wrap target job execution.
               $PATH_TO_DDT_CLIENT_SESSION_FILE" \
               ./geopm_test_integration.py -v TestIntegration.test_count`
```

Comment:
Isn't this use case broken with this PR?

PYTHONPATH isn't set to anything special in these cases.

Reply:
yes, these comments should be updated

Reply:
geopm_test_integration.py doesn't exist
The PYTHONPATH isn't set.

2 different examples showing how to do this via unittest discovery and by manually executing a single py file would be helpful.